### PR TITLE
Fix issue where editing welcome widget message wouldn't update immedi…

### PR DESCRIPTION
**Pull request by tibetsprague**
…ately

Also make it so links work in the welcome message, and you can use markdown to style the text!

Fixes https://github.com/Hylozoic/hylo-evo/issues/921

### CODE REVIEW

File: src/components/Widget/WelcomeWidget/WelcomeWidget.js [hunk 1]
```diff
@@ -12,10 +13,11 @@ export default class WelcomeWidget extends Component {
 
   render () {
     const { group, settings = {} } = this.props
+    const message = settings.text || `We're happy you're here! Please take a moment to explore this page to see what's alive in our group. Introduce yourself by clicking Create to post a Discussion sharing who you are and what brings you to our group. And don't forget to fill out your profile - so likeminded new friends can connect with you!`
     return (
       <div styleName='welcome-widget'>
         <h2>{settings.title || `Welcome to ${group.name}!`}</h2>
-        <p>{settings.text || `We're happy you're here! Please take a moment to explore this page to see what's alive in our group. Introduce yourself by clicking Create to post a Discussion sharing who you are and what brings you to our group. And don't forget to fill out your profile - so likeminded new friends can connect with you!`}</p>
+        <p dangerouslySetInnerHTML={{ __html: markdown(message) }} />
```
Comment (thomasgwatson) [inline src/components/Widget/WelcomeWidget/WelcomeWidget.js:20]: Are there alternatives to dangerouslySet, I thought they were pushing peeps away from using that?

**Comment by tibetsprague** [comment]
Hmm, probably. I just copied this from what we do in the group sidebar about text right now.

**Comment by thomasgwatson** [comment]
Ahh ok... well, something for the backlog :)

**Review by thomasgwatson — APPROVED** [review]

**Comment by tibetsprague** [comment]
@thomasgwatson actually i think this is something we should deal with. we are calling the markdown function from hylo-utils/text and it doesnt do any sanitizing! so i a going to add that there: https://github.com/Hylozoic/hylo-utils/blob/master/src/text.js

**Pull request merged**